public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2636,6 +2636,101 @@ Granting progress on the `party sheet` could happen when milestones or long-term
 >
 > `Supporting details` can be upgraded or downgraded based on filling the `progress` and `danger` clocks, respectively. The `danger` clock might be filled as a consequence, while the `progress` clock is filled each milestone and as the result of `long-term projects`. Ticks on the `progress` clock can be burned as momentum on a one-for-one basis.
 
+# Threat Level Extra | Extras
+
+**An alternate way to think adversity in Charge**
+
+\- By othelarian
+
+---
+
+Sometimes, ennemies are so powerful they have a direct effect over characters, hindering their abilityto fight back.
+
+Threat Level is an extra about adversity and how it can affect the character's dice rolls.
+
+## What is a Threat Level?
+
+While adversity doesn't affect directly the character, it's ok to use position and effect. But when it has an impact, it can be less obvious on how to deal with it and make it count mechanically speaking.
+
+`Threat Level` is a way to represent this.
+
+It takes the form of a value associated with the adversity. The higher the value, the most threatening it is.
+
+## How to define a Threat Level?
+
+A Threat Level has two main components:
+
+* A defined source of the `Threat Level`
+* A value associated with the source
+
+The source must be clearly identify and describe to let no room for player's interpretation. Also, the source must define how it generates the `Threat Level`.
```

Comment:
```suggestion
The source must be clearly identified so that it there's no room for the players' interpretation. Also, the source must define how it generates the `Threat Level`.
```